### PR TITLE
Parse a Figma design to create a Mobile UI part-1.

### PR DESCRIPTION
**Pull request by AbdusSattar-70**
In this milestone one, I have used the Figma design project template 2 to create a mobile version following below:

Added toolbar and homepage's first section,
Improved mobile responsiveness at 375px width and below,
used simple HTML and CSS and the npm package.
have created a tool toggler from one page to another page.
Replace png images with high-quality SVG images.

### CODE REVIEW

**Review by Zuhaib042 — COMMENTED** [review]
Hi @AbdusSattar-70 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project for re-review. fo that purpose you should have submitted the same pull request that you submitted for the first review. now I can't see the first review on your pull request.kindly after making the changes required by the first code reviewer, submit the same pull request for re-review 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by AbdusSattar-70** [comment]
> Hi @AbdusSattar-70 ,
> 
> While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
> 
> ## Invalid Code Review Request
> You have submitted a project for re-review. fo that purpose you should have submitted the same pull request that you submitted for the first review. now I can't see the first review on your pull request.kindly after making the changes required by the first code reviewer, submit the same pull request for re-review
> 
> Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _Invalid Code Review Request **does not count** into the code reviews limit._

@Zuhaib042 I don't know how to do it please give me a suggestion.

**Review by bienvenuushindi — APPROVED** [review]
# Approved 👏  

## Hi @AbdusSattar-70   👋, 

Well done implementing the requested changes #3 👏

### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

### Suggestion

- NOTE: **It's not good  to open a new PR when requesting for a re-review. Always use initial PR for all re-reviews. Please don't create  new PR next time.**

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @bienvenuushindi  in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  <link
+    href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@500&family=Poppins&family=Roboto:wght@700&display=swap"
+    rel="stylesheet" />
+</head>
+
+<body>
+  <!------------ header start---------------------- -->
+  <header>
+    <nav class="navbar-container">
+      <img class="branding-logo" src="./logo/Logo Placeholder.svg" alt="logo Placeholder" />
```
[the remark this answers is not in the document]
Comment (AbdusSattar-70) [reply src/index.html:27]: Thanks

**Pull request merged**